org.equicord.equibop.yml

```diff
@@ -0,0 +1,85 @@
+app-id: org.equicord.equibop
+
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '24.08'
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: *runtime-version
+
+command: startequibop
+separate-locales: false
+
+finish-args:
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.kde.StatusNotifierWatcher # Tray functionalities on KDE
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --device=all # Needed for GPU acceleration and the webcam
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download # This and the above two are used for drag-n-drop, and download managing
+  - --filesystem=xdg-run/pipewire-0 # Pipewire interfacing
+  - --filesystem=xdg-run/speech-dispatcher # For TTS and VcNarrator
+  - --filesystem=~/.steam # Needed for SteamOS integration, as the XDG OpenURL portal doesn't work properly in Game Mode. We only need ~/.steam/steam.pipe but Flatpak can't correctly mount just that: 'File "/home/deck/.steam/steam.pipe" has unsupported type 0o10000'
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons # This is used to show the correct cursor on Wayland
+
+modules:
+  - name: Equibop
+    buildsystem: simple
+    build-commands:
+      - chmod +x Equibop.AppImage
+      - ./Equibop.AppImage --appimage-extract
+      - |
+        if [ -d "squashfs-root/resources/bun/linux-x64" ]; then
+          install -Dm755 squashfs-root/resources/bun/linux-x64/bun-linux-x64/bun /app/bin/bun
+        else
+          install -Dm755 squashfs-root/resources/bun/linux-arm64/bun-linux-aarch64/bun /app/bin/bun
+        fi
+      - desktop-file-edit --set-key="Exec" --set-value="startequibop %U" --set-icon=$FLATPAK_ID
+        squashfs-root/equibop.desktop
+      - install -D squashfs-root/equibop.svg /app/share/icons/hicolor/scalable/apps/org.equicord.equibop.svg
+      - install -D squashfs-root/.DirIcon /app/share/icons/hicolor/256x256/apps/org.equicord.equibop.png
+      - install -D squashfs-root/equibop.desktop /app/share/applications/org.equicord.equibop.desktop
+      - install -Dm755 startequibop /app/bin/startequibop
+      - sed -i 's|<launchable type="desktop-id">equibop.desktop</launchable>|<launchable
+        type="desktop-id">org.equicord.equibop.desktop</launchable>|' org.equicord.equibop.metainfo.xml
+      - sed -i 's|</description>|</description>\n  <developer id="org.equicord">Equicord</developer>|'
+        org.equicord.equibop.metainfo.xml
```

Comment:
don't use sed to modify files. it has no failure mode and it's too obscure. For the desktop file look at `rename-desktop-file instead`. And for the icon `rename-icon`. or a patch.